boosters/node-beginner/src/app.js

```diff
@@ -5,13 +5,17 @@ const bodyParser = require("body-parser");
 
 const app = express();
 
-app.listen(process.env.port || 8080, () => {
-  console.log(`Server ready. Listening at http://localhost:8080`);
-});
+if (process.env.NODE_ENV === "development") {
```

Comment:
Why restrict this to development?

Reply:
I  don't  want my server listening for incoming requests when testing. If there were more environments other than development and test `!==="test"` makes more sense, but I think the only environments we'll have are development and test. I'll  change it to `!===test` thought